README.md

```diff
@@ -0,0 +1,78 @@
+# filecoin-docs
+Filecoin Docs
+
+
+<!--TO-DO: where can I get assets for Filecoin to replace IPFS logo and Build, Made, Project Icons?-->
+
+![IPFS documentation logo.](ipfs-docs-header.png)
+[![Build status icon.](https://img.shields.io/circleci/project/github/ipfs/ipfs-docs/master.svg?style=flat-square)](https://circleci.com/gh/ipfs/ipfs-docs)
+[![Made by icon.](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](https://protocol.ai/)
+[![Project icon.](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+
+This repository organizes overall documentation issues across the Filecoin project.
+
+**If you'd just like to read the Filecoin docs, we recommend the website version of the info contained in this repo! https://docs.filecoin.io/**
+
+## Get involved
+
+<!--TO-DO: do we want to link to this same bounty page or is there a Filecoin specific one?-->
```

Comment:
There is no active bounty program, so let's comment out everything related to bounties (part of line 20, plus everything in the Bounties section beginning line 26).